rubin_sim/data/rs_download_sky.py

```diff
@@ -10,6 +10,19 @@
 
 # Hack it up to find the filenames ending with .h5
 class MyHTMLParser(HTMLParser):
+    """HTML parser class that uses the HTMLParser to parse a starttag.
+
+    See Also
+    --------
+    html.parser.HTMLParser
+
+    Examples
+    --------
+    To define the coordinate of the M31 galaxy:
+
+    >>> parser = MyHTMLParser()
```

Comment:
What's M31 got to do with HTML? ;-)

Reply:
this was a copy paste from an example that I did not update before committing